plugins/text-freetype2/text-functionality.c

```diff
@@ -155,7 +201,9 @@ void fill_vertex_buffer(struct ft2_source *srcdata)
 
 	FT_UInt glyph_index = 0;
 
-	uint32_t dx = 0, dy = srcdata->max_h, max_y = dy;
+	uint32_t current_line_index = 0;
+	uint32_t dx = get_dx_for_line(srcdata, 0);
+	uint32_t dy = srcdata->max_h, max_y = dy;
```

Comment:
Style:  Separate the declarations for `dy` and `max_y` into two separate statements:
```
uint32_t dy = srcdata->max_h;
uint32_t max_y = dy;
```

Reply:
As you can see in deleted line, there were three declarations in one line that I divided into two lines, but I can divide into three if you want.

Reply:
There were, and if Jim ultimately wants to make multiple declarations on one line for whatever reason, that's his prerogative as maintainer.  For the purposes of my reviews though, if you've changed that line for a functional purpose, you might as well change it to better comply with the style guidelines.